docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,187 @@
+# Getting started with supercomputing at CSC
+
+You have signed up for your CSC account and first project, and are now ready to
+scale up your computing! This page provides guidance for getting started with
+using our HPC resources.
+
+It is recommended that new users complete the CSC Computing Environment course,
+which provides an in-depth introduction to CSC services. The next teaching
+instance can be found in the
+[CSC training calendar](https://csc.fi/en/trainings/training-calendar/).
+A [self-learning implementation](https://csc.fi/en/training-calendar/csc-computing-environment-self-learning/)
+is also available. The
+[practical exercises](https://csc-training.github.io/csc-env-eff/) are
+accessible without having to sign up, and quite useful by themselves.
```

Comment:
```suggestion
[course materials](https://csc-training.github.io/csc-env-eff/) are
accessible without having to sign up, and are very useful by themselves.
```

Reply:
Us saying ourselves that "they are useful" feels a bit weird. Maybe we could rather describe how it's useful? 

Maybe "... can be accessed without login via our GitHub page (link). The materials are self-learning ready, i.e. you can read through the materials on your own time and even do the exercises (using your own CSC account and project); solutions are provided. " or something along those lines?

Reply:
Yes, I agree, but on the other hand I would not like to extend the page too much anymore. It is already becoming much longer than I initially envisioned.

Reply:
I think that since that since the option of checking out just the exercises without signing up for the course is offered as an additional resource rather than something we would strongly want the users to do, it is not necessary to motivate why the material is useful. Also I would imagine that especially new users would be more or less inclined to trust CSC docs when it says that something is useful : )

Reply:
"The [exercise materials](https://csc-training.github.io/csc-env-eff/) are accessible without having to sign up, and are very useful by themselves, since they provide practical examples of working in our computing environment."